### DISCUSSION TITLE
Building a Customer Explorer Page

### DISCUSSION THREAD

**Discussion by jbrace02** [ordinals/ord#1851] [Q&A]
Hello, 

I was wanting to create my own Ordinal Explorer page and was curious what the steps would be need to be started. 

I have a fully indexed node.

I was trying to do some research and it seemed like I would need https://github.com/bitpay/bitcore and connect it to node. 

Wasn't sure if there was an easier way or how the current explorer page works.

**Comment by hydren-crypto** [comment]
see the ord server: 


https://github.com/casey/ord/blob/938b5cc97d48d026fd9250eae661d1e87286b377/docs/src/guides/explorer.md

**Comment by jbrace02** [comment]
I have majority of project built I'm working on, but question I have is how can I discern between OP_Return data that contains Ordinal data and blocks that do not? 

Also, how is the data transcoded so it displays in proper format on website? 

Any help is appreciated!

**Comment by nammaki** [comment]
I don't think it even uses OP_RETURN 🤔
